src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -152,8 +152,8 @@ public static ProjectName parseProjectName(String name) throws ParseException {
     public static String parseDescription(String description) throws ParseException {
         requireNonNull(description);
         String trimmedDescription = description.trim();
-        if (!trimmedDescription.matches("[\\p{Alnum}][\\p{Alnum} ]*")) {
```

Comment:
I'm not sure about this 2, one you taken from address field right, and the original I take from project name.

Reply:
Yup! I took from address field because that allows for long and varied sentences. I think the one from project name allows only for 2 words